docs/setup_installation/azure/getting_started.md

```diff
@@ -352,7 +352,46 @@ Click on *Save* (7).
   </figure>
 </p>
 
-## Step 3: Adding a ssh key to your resource group
+## Step 3: Create an ACR Container Registry
+
+The Hopsworks clusters deployed by [managed.hopsworks.ai](https://managed.hopsworks.ai) store their docker images in a registry container in your Azure account. To enable this you need to perform the following operations
+
+### Step 3.1: Create an ACR Container Registry
+
+Proceed to the Azure Portal and open the Resource Group that you want to use for [managed.hopsworks.ai](https://managed.hopsworks.ai). Click on *Create*, search for *Container Registery*, and click on *Create*. Then, make sure to choose the same Location as the one to deploy your cluser, choose a name for the registry and select **Premium** for the SKU. Then press *Review + create*. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/setup_installation/managed/azure/create-acr.png">
+      <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/azure/create-acr.png" alt="Create a Container Registry">
+    </a>
+    <figcaption>Create a Container Registery</figcaption>
+  </figure>
+</p>
+
+### Step 3.2: Give the Managed Identity access to the container registry
+
+Go to the managed identity created in [Step 2.2](#step-22-creating-a-user-assigned-managed-identity). Click on *Azure role assignments* in the left column. Click on *Add role assignment*. For the *Scope* select *Resource group* or *Subscription* depending on your preference. Select the *Role* *AcrPull* and click on *Save*.
+
+Repeat the same operation with the following roles:
+
+* *AcrPull*
+* *AcrPush*
+* *AcrDelete*
+
+### Step 3.3: Setup retention policy 
+
+To prevent the registry from filling up with unnecessary images and artifacts you can enable a retention policy. A retention policy will automatically remove untagged manifests after a specified number of days. To enable a retention policy, go to the registry you created. Go to the **Retention (preview)** tab and set **Status** from **disabled** to **enabled**. Set the retention policy for 7 days as in the figure below, then press **save**.
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/setup_installation/managed/azure/acr-retention.png">
+      <img style="border: 1px solid #000" src="../../../assets/images/setup_installation/managed/azure/acr-retention.png" alt="ACR retention policy">
+    </a>
+    <figcaption>ACR retention policy</figcaption>
+  </figure>
+</p>
+
+## Step 4: Adding a ssh key to your resource group
```

Comment:
You need to update the links pointing to this.

Reply:
fixed